docs/troubleshooting.md

```diff
@@ -12,4 +12,28 @@ To connect on our servers you need a working yubikey.
 5) Check available debian package versions: `ll -trh /srv/local-apt-repository`
 6) Check if latest database migration was applied: `sudo -u backend psql entitlementcard` then `SELECT * from migrations;`
 7) Create admin/store manager accounts: `sudo -u backend /opt/ehrenamtskarte/backend/bin/backend create-admin <command>`. You can find example in [runConfigs](../.idea/runConfigurations)
-8) Check if webhook for the debian package installation worked: `journalctl -x --since "48 hours ago" | grep webhook` and find info about the debian package installation: `vim /var/log/apt/history.log`
\ No newline at end of file
+8) Check if webhook for the debian package installation worked: `journalctl -x --since "48 hours ago" | grep webhook` and find info about the debian package installation: `vim /var/log/apt/history.log`
+
+### Import EAK accepting stores
+
+Import of EAK accepting stores on prod and staging envs run on **every odd day** of the month.
+
+:white_check_mark: When the job completes successfully, the following log entries can be found in the system log:
+
+```
+root@entitlementcard-test ~ # grep eak-update-data /var/log/syslog
+2025-04-21T02:00:01.943911+00:00 entitlementcard-test.tuerantuer.org CRON[2514930]: (backend) CMD (/usr/local/bin/eak-update-data)
+2025-04-21T02:06:41.365867+00:00 entitlementcard-test.tuerantuer.org eak-update-data: Successfully imported EAK data
+```
+
+:x: If the job fails, a notification is sent to the @entitlementcard-cert channel:
+
+`[entitlementcard-test.tuerantuer.org] ERROR eak-update-data at Wed Apr 9 04:14:19 PM UTC 2025`
+
+The output of the job is not saved.
+In case of failure the import can be restarted manually:
+
+```
+root@entitlementcard-test ~ # su - backend
```

Comment:
🙃 i think we can easily use a one liner here
`sudo -u backend /opt/ehrenamtskarte/backend/bin/backend import`